ramls/request.json

```diff
@@ -84,6 +84,10 @@
           "type": "string",
           "readonly": true
         },
+        "callNumber" : {
```

Comment:
Why has this property been added in order for it to be deprecated? Was it missing from the schema definition.

Reply:
yes, it was missed initially. Request representation sets this field [here](https://github.com/folio-org/mod-circulation/blob/master/src/main/java/org/folio/circulation/domain/RequestRepresentation.java#L90)